### ISSUE TITLE
Problem with splitChannels

### ISSUE THREAD

**Issue by lovetann** [katspaugh/wavesurfer.js#969]
I try to set splitChannels = true to see both channels in my audio files, but I only see flat lines. This example shows what I am after: https://wavesurfer-js.org/examples/

How do I go about to achieve this?

Here is what I've tried:
```

render() {
	const waveOptions = {
	   splitChannels: true,     
           height: 256,
           waveColor: 'darkorange',
	   progressColor: 'purple',
    };
    return (
		<div>
			<button onClick={this.handleTogglePlay} className="btn btn-primary btn-block">
					toggle play
		       </button>            
			<Wavesurfer
				  audioFile={"https://wavesurfer-js.org/example/split-channels/stereo.mp3"}
				  pos={this.state.pos}
				  options={waveOptions}
				  onPosChange={this.handlePosChange}
				  playing={this.state.playing}			                        
			/>
		</div>
      );
  }


```

**Comment by mspae** [comment]
Have you tried it with version 1.3.1?

**Comment by lovetann** [comment]
I have now, and it has solved the problem, - thanks!

Is there a way to mute one of the channels?

**Comment by thijstriemstra** [comment]
Fixed in 1.3.1.
